src/niscope/system_tests/test_system_niscope.py

```diff
@@ -24,11 +29,17 @@
 
 
 @pytest.fixture(scope='function')
-def session():
+def single_instrument_session():
     with niscope.Session('FakeDevice', False, True, 'Simulate=1, DriverSetup=Model:5164; BoardType:PXIe') as simulated_session:
         yield simulated_session
 
 
+@pytest.fixture(scope='function')
+def session():
```

Comment:
consider calling this `multi_instrument_session`

Reply:
done